editor/editor_node.cpp

```diff
@@ -5032,6 +5039,7 @@ EditorNode::EditorNode() {
 	p->add_child(tool_menu);
 	p->add_submenu_item(TTR("Tools"), "Tools");
 	tool_menu->add_item(TTR("Orphan Resource Explorer"), TOOLS_ORPHAN_RESOURCES);
+	tool_menu->add_item(TTR("Rebuild Documentation"), TOOLS_RELOAD_DOCS);
```

Comment:
I don't see the point of this feature.

Rebuilding documentation is only relevant when you made code changes and compiled them, and since that usually happens on the command line, it's much easier to run `--doctool` directly.

Reply:
Well, the idea (at the time) was to be able to have the doctool generate content for XML files in your project folder / plugin folders too. Combined with the ability to generate XML files from a Script, this would allow you to create in-engine documentation for the scripts in your project. And in a case like that, being able to easily rebuild stuff from the editor would be useful.

However, it's a fair point that all of that is predicated on features that aren't even implemented yet (script-based XML generation, generating docs from local files rather than just the godot repo).

I can go ahead and remove it from this PR.